src/index.ts

```diff
@@ -26,6 +35,17 @@ export function apply(ctx: Context, config: Config) {
 
   const api = new ChatGPT(ctx, config)
 
+  const getMapKey = (() => {
+    switch (config.conversationContext) {
+      case 'user':
+        return (session: Session) => session.uid
+      case 'channel':
+        return (session: Session) => session.cid
+      case 'both':
+        return (session: Session) => `${session.cid}:${session.uid}`
```

Comment:
Is it correct to diffre users with this?

Reply:
`${platform}:${channelId}:${userId}`